docs/reference/ccr/use-cases.asciidoc

```diff
@@ -0,0 +1,121 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-use-cases]]
+=== {ccr-cap} use cases
+++++
+<titleabbrev>Use cases</titleabbrev>
+++++
+
+Use {ccr} to enable several use cases within the Elastic Stack:
+
+* <<ccr-disaster-recovery,Disaster recovery>> in case a primary cluster fails,
+with a secondary cluster serving as a hot backup
+* <<ccr-data-locality,Data locality>> to serve reads locally and reduce costly latency
+* <<ccr-centralized-reporting,Centralized reporting>> for preventing search load from interfering with indexing by offloading search to a secondary cluster
+
+Watch the
+https://www.elastic.co/webinars/replicate-elasticsearch-data-with-cross-cluster-replication-ccr[{ccr} webinar] to learn more about the following use cases.
+Then, <<ccr-getting-started,set up {ccr}>> on your local machine and work
+through the demo from the webinar.
+
+[[ccr-disaster-recovery]]
+==== Disaster recovery and high availability
+Disaster recovery provides your mission-critical applications with the
+tolerance to withstand datacenter or region outages. This use case is the
+most common deployment of {ccr}. You can configure clusters in different
+architectures to support disaster recovery and high availability:
+
+* <<ccr-single-datacenter-recovery>>
+* <<ccr-multiple-datacenter-recovery>>
+* <<ccr-chained-replication>>
+* <<ccr-bi-directional-replication>>
+
+[discrete]
+[[ccr-single-datacenter-recovery]]
+===== Single disaster recovery datacenter
+In this configuration, data is replicated from the production datacenter to the
+disaster recovery datacenter. Because the follower indices replicate the leader
+index, {es} can use the disaster recovery datacenter if the production
+datacenter is unavailable.
+
+image::images/ccr-arch-disaster-recovery.png[Production datacenter that replicates data to a disaster recovery datacenter]
+
+[discrete]
+[[ccr-multiple-datacenter-recovery]]
+===== Multiple disaster recovery datacenters
+You can replicate data from one datacenter to multiple datacenters. This
+configuration provides both disaster recovery and high availability, ensuring
+that data is replicated in two datacenters if the primary datacenter is down
+or unavailable.
+
+In the following diagram, data from Datacenter A is replicated to
+Datacenter B and Datacenter C, which both have a read-only copy of the leader
+index from Datacenter A.
+
+image::images/ccr-arch-multiple-dcs.png[Production datacenter that replicates data to two other datacenters]
```

Comment:
The linear arrangement makes the distinction between this and the chained version less obvious. I'd stack the two DR datacenters so you don't have to trace the lines.

Reply:
Agreed. I'm going to redo all of those diagrams for color and size consistency, and will update this one to show the flow better.